src/robot/libdocpkg/htmlwriter.py

```diff
@@ -113,8 +115,9 @@ def _yield_header_targets(self, introduction):
                 yield match.group(2)
 
     def _escape_and_encode_targets(self, targets):
-        return NormalizedDict((html_escape(key), self._encode_uri_component(value))
-                              for key, value in targets.items())
+        return NormalizedDict(
+            (html_escape(key), self._encode_uri_component(value))
+            for key, value in targets.items())
```

Comment:
Why was this change done? In my opinion the old formatting was better. Was the line perhaps over 80 chars? If yes, that's fine as I've anyway been thinking that we should make the line length limit 88 chars.

If the formatting would be changed, then the opening and closing parenthesis should be balanced like

```python
return NormalizedDict(
    params,
    more_params
)
```
not like
```python
return NormalizedDict(
    params,
    more_params)
```